server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1397,6 +1377,63 @@ public boolean performConsumerUpdates(ConsumerDTO updated, Consumer toUpdate,
         return changesMade;
     }
 
+    private boolean updateEnvironment(ConsumerDTO updated, Consumer toUpdate) {
+        boolean changesMade = false;
+        boolean environmentFound = false;
+        Environment e = null;
+        String environmentId = updated.getEnvironment() == null ? null : updated.getEnvironment().getId();
+        String environmentName = updated.getEnvironment() == null ? null : updated.getEnvironment().getName();
+
+        if (environmentName != null && environmentId == null) {
+            List<Environment> environment = environmentCurator.listForOwnerByName(toUpdate.getOwner(),
+                environmentName).list();
+
+            if (!environment.isEmpty() && environment.size() == 1) {
+                e = environment.get(0);
+                environmentId = environment.get(0).getId();
+                environmentFound = true;
+            }
+            else {
+                throw new NotFoundException(i18n.tr(
+                        "Environment with name \"{0}\" could not be found.", environmentName));
```

Comment:
Indentation is too long here. If checkstyle is complaining, drop the i18n.tr down a line with it. Same with the identical line below.

Reply:
No checkstyle complaints.

Reply:
I meant if checkstyle complains about it not being indented enough. The indentation check is pretty spotty at times.